clover/launch/led.launch

```diff
@@ -22,7 +22,7 @@
         <param name="fade_period" value="0.5"/>
         <param name="rainbow_period" value="5"/>
         <!-- events effects table -->
-        <rosparam param="notify" if="$(arg led_notify)">
+        <rosparam param="notify" if="$(eval led_notify == 'all')">
```

Comment:
Maybe we could have a set of default notifications and append to it somehow?

Reply:
Don't completely get what you mean.